src/ray/raylet/node_manager.h

```diff
@@ -322,11 +322,11 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   ///
   /// \param actor_id The actor ID of the actor whose state was updated.
   /// \param data Data to publish.
-  /// \param failure_callback An optional callback to call if the publish is
+  /// \param tolerate_failure If tolerate update failure, if not will call Log Fatal
+  /// when update failed.
```

Comment:
below line should be removed.

Reply:
done

Reply:
i don't think `concurrent_update` is correct. NodeManager's logic here is check whether the update is success. Under certain conditions, update failed is unacceptable and the process will abort.